04 Hidden Search Widget/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css"
+        integrity="sha512-+4zCK9k+qNFUR5X+cKL9EIR+ZOhtIloNl9GIKS57V1MyNsYpYcUrUeQc9vNfzsWfV28IaLL3i96P9sdNyeRssA=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="style.css">
+    <title>Hidden serach widget</title>
+</head>
+
+<body>
+
+    <div class="find">
+        <input class="input" type="search" placeholder="What do you need?">
+        <button class="btn">
+            <i class="fas fa-search"></i>
+        </button>
+        <script src="script.js"></script>
```

Comment:
It will be better if the script will be loaded right before closing `</body>` tag or in the `<head>`. Try both if works.